client/app/widgets/Resources/index.jsx

```diff
@@ -110,28 +119,42 @@ export class Resources extends React.Component<Props, State> {
     });
   };
 
+  onClick = () => {
+    const { resources } = this.props;
+    this.setState((prevState: State) => ({
+      resourcesDisplayed: prevState.resourcesDisplayed + RESOURCES_PER_PAGE,
+      lastPage:
+        prevState.resourcesDisplayed + RESOURCES_PER_PAGE >= resources.length,
+    }));
+  };
+
   displayTags = () => {
-    const { checkboxes } = this.state;
+    const { checkboxes, resourcesDisplayed, lastPage } = this.state;
+    const { resources } = this.props;
     const filteredResources = this.filterList(checkboxes);
     return (
       <>
         <center className={css.marginTop}>
-          {`${filteredResources.length} ${I18n.t(
-            'navigation.resources',
-          ).toLowerCase()}`}
+          {`${Math.min(resourcesDisplayed, resources.length)} of ${
```

Comment:
As mentioned in the closed PR, we should put these in our locales. I wonder if we should be displaying this info here. Maybe it's worth moving this in an aria-label for the Load More button.